src/ontology/edits/oeo-physical.omn

```diff
@@ -672,98 +613,99 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/273",
+         and (<http://openenergy-platform.org/ontology/oeo/has_origin> some <http://openenergy-platform.org/ontology/oeo/OEO_00030004>)
     
     SubClassOf: 
-        <http://purl.obolibrary.org/obo/RO_0000091> some OEO_00000097
+        <http://purl.obolibrary.org/obo/RO_0000091> some <http://openenergy-platform.org/ontology/oeo/OEO_00000097>
     
     
-Class: OEO_00000034
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00000034>
 
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "A solar power unit is a power generating unit using solar power.",
         rdfs:label "solar power unit"
     
     SubClassOf: 
-        OEO_00000334
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000334>
     
     
-Class: OEO_00000035
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00000035>
 
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "A solar thermal power unit is a solar power unit that has solar thermal collectors as parts.",
         rdfs:label "solar thermal power unit"
     
     SubClassOf: 
-        OEO_00000034,
-        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00000387
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000034>,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some <http://openenergy-platform.org/ontology/oeo/OEO_00000387>
     
     
-Class: OEO_00000036
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00000036>
 
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "A solid biomass powerplant is a biomass powerplant having an aggregate of solid biomass power units as its power generating units.",
         rdfs:label "solid biomass powerplant"
     
     SubClassOf: 
-        OEO_00000073
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000073>
     
     
-Class: OEO_00000037
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00000037>
 
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "A solid biomass power unit is a biomass power unit using solid biomass as fuel.",
         rdfs:label "solid biomass power unit"
     
     SubClassOf: 
-        OEO_00000003,
-        uses some OEO_00000332
+        <http://openenergy-platform.org/ontology/oeo/OEO_00000003>,
+        <http://openenergy-platform.org/ontology/oeo/uses> some <http://openenergy-platform.org/ontology/oeo/OEO_00000332>
     
     
-Class: OEO_00000038
+Class: <http://openenergy-platform.org/ontology/oeo/OEO_00000038>
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Sulphur hexafluoride is a portion of matter with the chemical formula SF6. It can work as a potent greenhouse gas and has an anthropogenic origin.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Sulphur hexafluoride is a portion of matter with the chemical formula SF6. has a gaseous normal state of matter. It can work as a potent greenhouse gas and has an anthropogenic origin.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Sulphur hexafluoride is a portion of matter with the chemical formula SF6. It has a gaseous normal state of matter. It can work as a potent greenhouse gas and has an anthropogenic origin.",
```